samples/scanner/lb.yaml

```diff
@@ -0,0 +1,32 @@
+rules:
+  - name: rajtest
+    target: targettest
+    mode: whitelist
+    load_balancing_scheme: EXTERNAL
+    ip_protocol: ESP
+    ip_address: "14.15.9.13"
```

Comment:
Please remove any IPs(especially public ones) from samples and maybe use upper case "INSERT IP HERE"